Code/Source/URDF/FbxNode.cpp

```diff
@@ -12,37 +12,38 @@
 #include <fstream>
 #include <string>
 
-#include <AzCore/Debug/Trace.h>
+#include <AzCore/std/string/string.h>
+#include <AzCore/Console/Console.h>
 
 namespace ROS2
 {
     namespace Fbx
     {
-        std::string AnyToString(const std::any & a)
+        AZStd::string AnyToString(const std::any & a)
```

Comment:
I'm so sorry, I meant to delete this suggestion because I realized AZStd::string does not implement to_string, so no need to do the unnecessary conversion